cms/djangoapps/contentstore/views/tests/test_videos.py

```diff
@@ -608,6 +634,143 @@ def test_no_video_image(self):
             else:
                 self.assertEqual(response_video['course_video_image_url'], None)
 
+    @ddt.data(
```

Comment:
Also It'd be great if we could have  few comments on to-be-validated `ddt` data (i.e. telling here starts extension validation data, size validation date, aspect ratio validation data). This would make `ddt` structure more readable.